src/Utility.cc

```diff
@@ -183,6 +183,25 @@ namespace gz
       return out;
     }
 
+    /////////////////////////////////////////////
+    math::SphericalCoordinates::CoordinateType Convert(
+      const msgs::SphericalCoordinatesType &_sc)
+    {
+      switch (_sc)
+      {
+        case msgs::SphericalCoordinatesType::ECEF:
+          return math::SphericalCoordinates::CoordinateType::ECEF;
+        case msgs::SphericalCoordinatesType::GLOBAL:
+          return math::SphericalCoordinates::CoordinateType::GLOBAL;
+        case msgs::SphericalCoordinatesType::SPHERICAL:
+          return math::SphericalCoordinates::CoordinateType::SPHERICAL;
+        case msgs::SphericalCoordinatesType::LOCAL:
+          return math::SphericalCoordinates::CoordinateType::LOCAL;
+        case msgs::SphericalCoordinatesType::LOCAL2:
+          return math::SphericalCoordinates::CoordinateType::LOCAL2;
```

Comment:
This block returns a warning on Windows as we don't return anything if none of the previous conditions are met.